frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -38,7 +40,7 @@ interface IReviewProps {
 }
 
 export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
-  const formikProps = useFormikContext();
+  const formikProps = useFormikContext<any>();
```

Comment:
Is it possible to not use `any`?

Reply:
looks like I can remove it, will fix that now :)